src/topology/perfect.lean

```diff
@@ -211,3 +218,107 @@ begin
 end
 
 end kernel
+end basic
+
+section cantor_inj
+
+open function
+variables {α : Type*} [metric_space α] {C : set α} (hC : perfect C)
+include hC
+
+private lemma perfect.small_diam_aux (ε : ennreal) (ε_pos : 0 < ε) {x : α} (xC : x ∈ C) :
```

Comment:
```suggestion
variables {α : Type*} [metric_space α] {C : set α} (hC : perfect C) {ε : ℝ≥0∞}
include hC

private lemma perfect.small_diam_aux (ε_pos : 0 < ε) {x : α} (xC : x ∈ C) :
```
Same for the other ones

Reply:
is there a locale I need to open for this notation? it is not working for me.